### DISCUSSION TITLE
Feature Request: Addition of a Tally Ho Connector

### DISCUSSION THREAD

**Discussion by 0xzoz** [wevm/wagmi#1049] [Ideas]
[Tally Ho](https://tally.cash/) is an open-source community-run wallet incubated by the team at [Thesis](https://thesis.co/) and community contributors.

As we near for the release of the DAO, It is essential to have the ability to put the brand front and center in applications within the ecosystem. We have had numerous projects using the awesome wagmi framework and want to use Tally Ho as a connector. This includes Pool Together and Rainbow Wallet. While Tally Ho is already added as an injected option, it makes sense to give applications the option to show Tally Ho even if it isn't installed. Having a community run and operated wallet embodies the web3 movement and the wagmi ethos. 

I have done most of the work, which I will link in a separate draft pull request after sufficient testing but I wanted to start the conversation here to get feedback, concerns and thoughts 🐶

**Comment by tmm** [comment]
Let's sync up to chat. Can you drop a note to dev@wagmi.sh?

**Reply by 0xzoz** [reply]
sure 👍
